shared/src/commonTest/kotlin/co/nimblehq/blisskmmic/domain/usecase/LogInUseCaseTest.kt

```diff
@@ -0,0 +1,78 @@
+package co.nimblehq.blisskmmic.domain.usecase
+
+import co.nimblehq.blisskmmic.domain.model.Token
+import co.nimblehq.blisskmmic.domain.repository.UserRepository
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.catch
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.test.runTest
+import org.kodein.mock.*
+import org.kodein.mock.tests.TestsWithMocks
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.fail
+
+@ExperimentalCoroutinesApi
+class LogInUseCaseTest : TestsWithMocks() {
+
+    private val email = "email@mail.com"
+    private val password = "pass"
+
+    @Mock
+    lateinit var userRepository: UserRepository
+    @Fake
+    lateinit var token: Token
+
+    val logInUseCase by withMocks { LogInUseCaseImpl(userRepository) }
+
+    override fun setUpMocks() = injectMocks(mocker)
+
+    @BeforeTest
+    fun setUp() {
+        mocker.reset()
+    }
+
+    @Suppress("MaxLineLength")
+    @Test
+    fun `Given LogInRepository returns success When LogInUseCase calls login Then it should return correct object`() = runTest {
```

Comment:
@blyscuit FYI, as per the [Android's test case naming convention](https://nimblehq.co/compass/development/android/testing/#test-case-naming) from Compass, the convention should be  `<When> _ <Expect> _` instead. Hence, I suggest you to refactor the test case naming here to match with our Android convention and we can also apply for the remaining test cases 🙏

Reply:
Fixed in https://github.com/blyscuit/kmm-swiftUI-IC/pull/53/commits/987050331b2b65faaab909760d63a88ad8f83180